cni/network/invoker_azure.go

```diff
@@ -75,8 +75,8 @@ func (invoker *AzureIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, er
 
 	defer func() {
 		if err != nil {
-			if len(addResult.defaultInterfaceInfo.IPConfigs) > 0 {
-				if er := invoker.Delete(&addResult.defaultInterfaceInfo.IPConfigs[0].Address, addConfig.nwCfg, nil, addConfig.options); er != nil {
+			if len(addResult.interfaceInfo[0].IPConfigs) > 0 {
+				if er := invoker.Delete(&addResult.interfaceInfo[0].IPConfigs[0].Address, addConfig.nwCfg, nil, addConfig.options); er != nil {
```

Comment:
Are we sure the first added interface is the default one ? 
In case of multiple adds calls as well  (trying to understand this better.)

Reply:
There is only 1 interface added to `IPAMAddResult` in this function.